docs/moz-addon-manager.md

```diff
@@ -96,94 +96,57 @@ at a URL like `localhost:3000`.
 diff --git a/browser/base/content/browser-addons.js b/browser/base/content/browser-addons.js
 --- a/browser/base/content/browser-addons.js
 +++ b/browser/base/content/browser-addons.js
-@@ -711,16 +711,11 @@ var LightWeightThemeWebInstaller = {
-     let uri;
-     try {
-       uri = makeURI(srcURIString);
-     } catch (e) {
-       // makeURI fails if srcURIString is a nonsense URI
+@@ -664,12 +664,11 @@ var LightWeightThemeWebInstaller = {
+   },
+ 
+   _isAllowed(principal) {
+-    if (!principal || !principal.URI || !principal.URI.schemeIs("https")) {
++    if (!principal || !principal.URI) {
        return false;
      }
-
--    if (!uri.schemeIs("https")) {
--      return false;
--    }
--
+ 
 -    let pm = Services.perms;
--    return pm.testPermission(uri, "install") == pm.ALLOW_ACTION;
+-    return pm.testPermission(principal.URI, "install") == pm.ALLOW_ACTION;
 +    return true;
-   }
- };
+   },
+ 
+   _shouldShowUndoPrompt(principal) {
 diff --git a/toolkit/mozapps/extensions/AddonManager.jsm b/toolkit/mozapps/extensions/AddonManager.jsm
 --- a/toolkit/mozapps/extensions/AddonManager.jsm
 +++ b/toolkit/mozapps/extensions/AddonManager.jsm
-@@ -64,16 +64,17 @@ var PREF_EM_CHECK_COMPATIBILITY = MOZ_CO
-
- const VALID_TYPES_REGEXP = /^[\w\-]+$/;
-
- const WEBAPI_INSTALL_HOSTS = ["addons.mozilla.org", "testpilot.firefox.com"];
+@@ -57,7 +57,7 @@ const WEBAPI_INSTALL_HOSTS = ["addons.mo
  const WEBAPI_TEST_INSTALL_HOSTS = [
    "addons.allizom.org", "addons-dev.allizom.org",
    "testpilot.stage.mozaws.net", "testpilot.dev.mozaws.net",
-   "example.com",
-+  "localhost",
+-  "example.com",
++  "example.com", "localhost",
```

Comment:
does this part need to be updated? or if the point is to update the order, should we keep it on its own line as it was before?

Reply:
I took the patch "as is" from my setup, it seems they have changed the formatting of this list..

Reply:
ah well now I understand what you meant, sorry yeah you're right, I should not have changed that.

Reply:
@willdurand  - were you going to revert your change here? i wasn't sure...

Reply:
or rather move to its own line?